uniffi-bindgen/src/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    uniffi::uniffi_bindgen_main();
+}
```

Comment:
im not completely sure if we should keep this or not, it definitely makes development have less friction, but for CI we should be using the generator image, because otherwise we would have to keep track of the generator versions manually

Reply:
Yeah, that's a good point. So I overlooked the image and went the official route. With the generator image, this isn't needed but I'm creating Python bindings on the fly and it would be nice not to pull the image for running tests.
So I'm voting for leaving this, but it's not a strong opinion

Reply:
since we're not going to be mixing and matching on CI, i think this is fine to keep, since it allows us to run tests a lot easier locally